tools/cldr-unittest/src/org/unicode/cldr/unittest/TestPluralRuleGeneration.java

```diff
@@ -102,43 +115,162 @@ public void TestEFixedDecimal() {
             throw new ICUException(e);
         }
         Object[][] checkItems = {
+            // compare each group of items against one another, where a group is bounded by {},
+            // source,  f,      v,  e,  plural (, skipString — if toString doesn't roundtrip)?
```

Comment:
nit: Should this comment be `// source, n, ...` instead of `// source, f, ...` ?